llvm/lib/Target/ARM/ARMBaseInstrInfo.cpp

```diff
@@ -6979,7 +6979,7 @@ bool ARMPipelinerLoopInfo::tooMuchRegisterPressure(SwingSchedulerDAG &SSD,
 
   auto &P = RPTracker.getPressure().MaxSetPressure;
   for (unsigned I = 0, E = P.size(); I < E; ++I)
```

Comment:
Can you add these RegClasses as excluded:
```
    // Exclude some Neon register classes.
    if (I == ARM::DQuad_with_ssub_0 || I == ARM::DTripleSpc_with_ssub_0 ||
        I == ARM::DTriple_with_qsub_0_in_QPR)
      continue;
```

Reply:
Thanks, Done!